### PR TITLE
READY FOR REVIEW: renamed binding to link everywhere

### PR DESCRIPTION
**Pull request by mikezupper**
all binding references were renamed to link.

### CODE REVIEW

File: crates/wascc-host/src/auth.rs [hunk 1]
```diff
@@ -194,7 +194,7 @@ mod test {
         let target = WasccEntity::Capability {
             contract_id: "wascc:keyvalue".to_string(),
             id: "Vxxx".to_string(),
-            binding: "default".to_string(),
+            link: "default".to_string(),
```
Comment (autodidaddict) [inline crates/wascc-host/src/auth.rs:197]: This should be `link_name` here.

File: crates/wascc-host/src/capability/native_host.rs [hunk 2]
```diff
@@ -109,7 +109,7 @@ impl Handler<Initialize> for NativeCapabilityHost {
         let entity = WasccEntity::Capability {
             id: state.cap.claims.subject.to_string(),
             contract_id: state.descriptor.id.to_string(),
-            binding: state.cap.binding_name.to_string(),
+            link: state.cap.link_name.to_string(),
```
Comment (autodidaddict) [inline crates/wascc-host/src/capability/native_host.rs:112]: Field name should be `link_name` on the WasccEntity::Capability variant

**Review by autodidaddict — CHANGES_REQUESTED** [review]

**Review by brooksmtownsend — APPROVED** [review]
LGTM

**Comment by autodidaddict** [comment]
Same as with the other PR, I think it makes sense to wait until #14 is done and then re-apply this change to main rather than attempting the merge conflict resolution since that PR basically makes changes everywhere.

**Comment by autodidaddict** [comment]
@mikezupper if you want you can try and resolve the conflicts here, or I can just try and do a by-hand merge offline. I tried to get git to do this for me automatically and it laughed at my weakness.

**Comment by mikezupper** [comment]
i will recreate a new PR with these changes. Too many files have been renamed or altered beyond GITs ability to reconcile the differences.

**Pull request closed without merging**